File: frontend/lib/features/transport/data/models/transport_status.dart
enum TransportStatus {
  active('active', 'Active'),
  inactive('inactive', 'Inactive');

  const TransportStatus(this.apiValue, this.label);

  final String apiValue;
  final String label;

  static TransportStatus fromApiValue(String value) => TransportStatus.values.firstWhere((s) => s.apiValue == value);
}

File: backend/app/Enums/TransportStatus.php
<?php

namespace App\Enums;

/**
 * Shared by vehicles, drivers and routes - "inactive" keeps the record (and
 * its history) but takes it out of every picker.
 */
enum TransportStatus: string
{
    case Active = 'active';
    case Inactive = 'inactive';
}
